core/platform/models.py

```diff
@@ -33,9 +33,23 @@ def import_models(cls):
 
 
 class _Gae(_Platform):
+    """Google App Engine Model.
+    Super model of the Registry Model and implements all the
+    methods of Registry.
+    """
     @classmethod
     def import_models(cls, model_names):
+        """Imports and returns the storage models listed in model_names.
 
+        Args:
+            model_names: list(str). List of storage model names.
+
+        Returns:
+            tuple(returned_models): Tuple of Storage models.
```

Comment:
Every storage model here has base_model as their parent class and base_model has ndb.model as its parent,  so do we write tuple(BaseModel) or tuple(ndb.model)?